hubmap/startup_script.py

```diff
@@ -164,8 +153,19 @@ def create_genes(hdf_file: Path):
 
 
 def create_organs(hdf_file: Path):
-    cell_df = pd.read_hdf(hdf_file, "cell")
-    organs = list(cell_df["organ"].unique())
+    if hdf_file.stem == "codex":
+        organs_set = set({})
+        store = pd.HDFStore(hdf_file)
+        for key in store.keys():
+            cell_df = pd.read_hdf(hdf_file, key)
+            organs = list(cell_df["organ"].unique())
+            for organ in organs:
+                organs_set.add(organ)
```

Comment:
These three lines could be replaced with `organs_set.update(cell_df["organ"])`, no? Calling `.unique` is redundant since you're adding to a `set`, the call to `list` makes a Python list out of a NumPy array (wasting memory), and then you iterate over this list in Python rather than letting a lower-level iteration method handle this.